Appsilon/rhino
# Feat: Adds spell check to workflow

### Changes
Closes #215 

* Adds spell check in `.github/workflows/ci.yml`.
* Adds `inst/WORDLIST` for the accepted words that initially was detected by `spelling::spell_check_package()` as bad spelling, but are good words, e.g. Appsilon, RStudio, js, shinyapps, ui, and yml.

### How to test
* Please see https://github.com/Appsilon/rhino/actions/runs/3240738061 for a failing workflow and https://github.com/Appsilon/rhino/actions/runs/3240789835 for a successful workflow.
* Remove one of the words in `inst/WORDLIST` and the workflow should fail.
